.github/workflows/sync-to-aur.yml

```diff
@@ -0,0 +1,45 @@
+name: Sync package to AUR
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  sync:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Save ssh key
+        run: |
+          mkdir -p ~/.ssh
+          echo ${{ secrets.AUR_SSH_KEY }} > ~/.ssh/id_rsa
+          chmod 600 ~/.ssh/id_rsa
+
+      - name: Install and configure git
+        run: |
+          sudo apt-get install git
```

Comment:
`apt install` and it already runs as root

Reply:
No, I've tested `apt install` but it need su permission.